Makefile

```diff
@@ -38,16 +43,19 @@ prerequisites:
 
 oclint-examples:
 	set -o pipefail && \
+	xcodebuild -scheme VelocidiSDK -sdk iphonesimulator -workspace VelocidiSDK.xcworkspace COMPILER_INDEX_STORE_ENABLE=NO clean build && \
 	xcodebuild -scheme ObjcExample -sdk iphonesimulator -workspace VelocidiSDK.xcworkspace COMPILER_INDEX_STORE_ENABLE=NO clean build | xcpretty -r json-compilation-database --output compile_commands.json && \
-	oclint-json-compilation-database -exclude Pods -exclude build -- -report-type xcode -max-priority-3=15000
+	oclint-json-compilation-database -exclude Pods -exclude build -- -report-type xcode
```

Comment:
Does this command exit with error if there are warnings or errors in the report?

Reply:
Yes